src/app/services/sync/voyager.service.ts

```diff
@@ -77,19 +76,15 @@ export class VoyagerService {
   }
 
   public async createApolloClient() {
+    const urls = this.configService.config.backend;
     const options: DataSyncConfig = {
+      httpUrl: urls.serverUrl,
+      wsUrl: urls.wsServerUrl,
       conflictListener: new ConflictLogger(this.alertCtrl),
       fileUpload: true,
-      mutationCacheUpdates: taskCacheUpdates
+      mutationCacheUpdates: taskCacheUpdates,
     };
 
-    if (!this.openShift.hasSyncConfig()) {
-      // Use default localhost urls when OpenShift config is missing
-      options.httpUrl = this.openShift.getLocalServerUrl();
-      options.wsUrl = this.openShift.getWSLocalServerUrl();
-    } else {
-      options.openShiftConfig = this.openShift.getConfig();
-    }
     options.authContextProvider = await this.authService.getAuthContextProvider();
     const offlineClient = await createClient(options);
```

Comment:
> createClient is deprecated: use OfflineClient class directly:

Reply:
Waiting for @darahayes to come up and amaze us with new api :D